simapp/app.go

```diff
@@ -287,11 +287,8 @@ func NewSimApp(
 	// add keepers
 	accountsKeeper, err := accounts.NewKeeper(
 		appCodec,
-		runtime.NewKVStoreService(keys[accounts.StoreKey]),
+		runtime.NewEnvironment(runtime.NewKVStoreService(keys[accounts.StoreKey]), logger),
 		runtime.EventService{},
```

Comment:
we don't need this, the event service is as well in environement.

Reply:
on using environment.EventService, it is panicking with the following 
`panic: interface conversion: interface {} is nil, not types.Context [recovered]
        panic: interface conversion: interface {} is nil, not types.Context`
trying to debug this

Reply:
is this in a test?

Reply:
yes, accounts module tests were failing

Reply:
hmm we should be able to fix it or something is being used somewhere even after its removed